exasol/cloud_storage.py

```diff
@@ -0,0 +1,48 @@
+import pyexasol  # type: ignore
+
+
+_SETUP_SQL = [
+        "OPEN SCHEMA {schema!i}",
+        """
+--/
+            CREATE OR REPLACE JAVA SET SCRIPT IMPORT_PATH(...) EMITS (...) AS
+              %scriptclass com.exasol.cloudetl.scriptclasses.FilesImportQueryGenerator;
+              %jar {jar_path!r};
+/
+        """,
+        """
+--/
+        CREATE OR REPLACE JAVA SCALAR SCRIPT IMPORT_METADATA(...) 
+          EMITS (
+                filename VARCHAR(2000), 
+                partition_index VARCHAR(100), 
+                start_index DECIMAL(36, 0), 
+                end_index DECIMAL(36, 0)
+          ) AS
+          %scriptclass com.exasol.cloudetl.scriptclasses.FilesMetadataReader;
+          %jar {jar_path!r};
+/
+        """,
+        """
+--/
+        CREATE OR REPLACE JAVA SET SCRIPT IMPORT_FILES(...) EMITS (...) AS
+          %scriptclass com.exasol.cloudetl.scriptclasses.FilesDataImporter;
+          %jar {jar_path!r};
+/
+        """
+    ]
```

Comment:
Keep in mind though that `cleandoc` removes the last `\n` in multiline strings, so if required use the following:
`cleandoc(...) + '\n'`

```suggestion
import pyexasol  # type: ignore

from inspect import cleandoc


_SETUP_SQL = [
        "OPEN SCHEMA {schema!i}",
        cleandoc("""
        --/
            CREATE OR REPLACE JAVA SET SCRIPT IMPORT_PATH(...) EMITS (...) AS
              %scriptclass com.exasol.cloudetl.scriptclasses.FilesImportQueryGenerator;
              %jar {jar_path!r};
        /
        """),
        cleandoc("""
        --/
        CREATE OR REPLACE JAVA SCALAR SCRIPT IMPORT_METADATA(...) 
          EMITS (
                filename VARCHAR(2000), 
                partition_index VARCHAR(100), 
                start_index DECIMAL(36, 0), 
                end_index DECIMAL(36, 0)
          ) AS
          %scriptclass com.exasol.cloudetl.scriptclasses.FilesMetadataReader;
          %jar {jar_path!r};
        /
        """),
        cleandoc("""
        --/
        CREATE OR REPLACE JAVA SET SCRIPT IMPORT_FILES(...) EMITS (...) AS
          %scriptclass com.exasol.cloudetl.scriptclasses.FilesDataImporter;
          %jar {jar_path!r};
        /
        """)
    ]
```